animation.go

```diff
@@ -22,11 +22,11 @@ var (
 // Animation represents an animated element within a Fyne canvas.
 // These animations may control individual objects or entire scenes.
 type Animation struct {
-	AutoReverse bool
```

Comment:
This is a change to a public API and will break any instances of &Animation{} without field indexes used.
Do we want to do that? I know the API is not final yet - but still worth noting the break.

Reply:
Well, technically adding a new field as a feature down the line will also break it (as you need to give a value to each field that way). I don't personally think it is a huge issue with this breakage, but I will let you decide :)

Reply:
Quite right, I hadn't thought of that.
I still think we would need a way to mark this as "not alphabetical for x reason" otherwise later refactors and/or reviews could reorder or reject.